### PR TITLE
New Pull request for activity

### PR DESCRIPTION
**Pull request by sakther**
Please discard previous pull request

---
This change is [Reviewable](https://reviewable.io/reviews/md2korg/cerebralcortex-dataanalysis/24)

### CODE REVIEW

File: core/signalprocessing/gravity_filter/gravityFilter.py [hunk 1]
```diff
@@ -30,13 +30,15 @@
 from cerebralcortex.core.datatypes.datastream import DataStream
 
 
-def gravityFilter_function(accl_stream, gyro_stream, sampling_freq=25.0, is_gyro_in_degree=True):
+def gravityFilter_function(accl_stream, gyro_stream, sampling_freq=25.0,
+                           is_gyro_in_degree=True):
```
Comment (anandatirtha) [inline core/signalprocessing/gravity_filter/gravityFilter.py]: fix hardcoding

File: core/signalprocessing/gravity_filter/gravityFilter.py [hunk 2]
```diff
@@ -53,7 +55,8 @@ def gravityFilter_function(accl_stream, gyro_stream, sampling_freq=25.0, is_gyro
     Acc_sync_filtered = []
```
Comment (anandatirtha) [inline core/signalprocessing/gravity_filter/gravityFilter.py]: fix hardcoding 0.4

File: core/signalprocessing/gravity_filter/gravityFilter.py (hunk 1 above)
Comment (sakther) [reply core/signalprocessing/gravity_filter/gravityFilter.py]: Fixed

File: core/signalprocessing/gravity_filter/gravityFilter.py (hunk 2 above)
Comment (sakther) [reply core/signalprocessing/gravity_filter/gravityFilter.py]: Fixed

**Comment by sakther** [comment]
fixed the stream names

**Comment by sakther** [comment]
Hi Ananda,
I have fixed the requested changes. Please check.

Regards,
Sayma

**Comment by sakther** [comment]
generate a new pull request

**Pull request closed without merging**